FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/report_builder.c

```diff
@@ -334,7 +451,7 @@ eReportBuilderStatus eGenerateJsonReport( char * pcBuffer,
     char * pcCurrentWritePos = pcBuffer;
     uint32_t ulRemainingBufferLength = ulBufferLength, bufferWritten;
```

Comment:
Simplify this for the reader by declaring `bufferWritten` on a separate line, i.e.:

```c
uint32_t ulRemainingBufferLength = ulBufferLength;
uint32_t bufferWritten;
```